packages/app/src/files-manager/components/files-manager/files-manager.component.tsx

```diff
@@ -58,11 +58,16 @@ const FilesManager: FunctionComponent<FilesManagerProps> = ({
           ...element,
           size: free,
         }
-      }
-      if (element.type === DiskSpaceCategoryType.UsedSpace) {
+      } else if (element.type === DiskSpaceCategoryType.UsedSpace) {
+        return {
+          ...element,
+          size: reservedSpace,
```

Comment:
Is it ok? Shouldn't it be `size: usedUserSpace`?

Reply:
To be precise
UsedSpace = `usedUserSpace` - sum of all music file

Reply:
Thanks for the comment! Thanks to it I realized that we shouldn't show used space. Instead, we should show Music and system. Changed in [2f838ad](https://github.com/mudita/mudita-center/pull/915/commits/2f838ad5888146b962ea0c53a4419615d2c45ec8)